packages/front-end/components/Experiment/VariationsTable.tsx

```diff
@@ -373,10 +414,19 @@ const VariationsTable: FC<Props> = ({
                             return (
                               <td key={j} className="px-4 py-1">
                                 <div className="d-flex justify-content-between">
-                                  <div>
+                                  <a
```

Comment:
Personal preference, but feels like this should be an edit button / pencil. Otherwise it's a lot of blue text under each changeset in the variations table.